wagtailio/packages/templates/packages/packages_page.html

```diff
@@ -0,0 +1,83 @@
+{% extends "base.html" %}
+
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block content %}
+
+    <header class="page-heading">
+        <h1 class="page-heading__heading">{{ page.title }}</h1>
+        <div class="page-heading__introduction">
+            Wagtail third-party packages
+            {% if value.subtitle %}{{ value.subtitle }}{% endif %}
+        </div>
+    </header>
+
+    <div class="features" id="additional">
+        <div class="features__container">
+            <h2 class="features__heading">{{ value.title }}</h2>
+            {% for grid in grids %}
```

Comment:
This looks like a pretty long page in the screenshot. I wonder if a table of contents or a select with grids that jumps to the selected grid may help navigation. Might be one to tackle after this is on staging/live and we see it in action

Reply:
Good idea! 

I was thinking of a client side search that filters the content on the page. But then you need to know where you are looking for.

Reply:
Done, I implemented your suggestion.